motoko/threshold-schnorr/README.md

```diff
@@ -57,14 +50,12 @@ This tutorial will use the Motoko version of the canister:
 cd examples/motoko/threshold-schnorr
 dfx start --background
 npm install
```

Comment:
The README mentions `npm install` but the required `package.json` does not exist: calling `npm install` results in an error.

In the Github workflow this issue is circumvented by installing the required package manually with `npm install @noble/curves`.

We should either update the README, or add a `package.json` (and lock file). As far as I can see, many other examples in the repo do have a `package.json` (and also the respective `package-lock.json`).

Reply:
I think installing the dependency directly is currently okay because 1) it's the only `npm` dependency and 2) it's a test dependency. I removed most `npm install`s from READMEs though and added different scripts for deployment and testing. PTAL.

Reply:
👍